### PR TITLE
adding timezone support to logparser timestamps

### PR DESCRIPTION
**Pull request by sebito91**
- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/logparser/README.md [hunk 1]
```diff
@@ -7,15 +7,17 @@ regex patterns.
 ### Configuration:
 
 ```toml
-[[inputs.logparser]]
```
Comment (danielnelson) [inline plugins/inputs/logparser/README.md]: Keep this line

File: plugins/inputs/logparser/README.md [hunk 2]
```diff
@@ -28,13 +30,27 @@ regex patterns.
...
+    
+	## Name of the outputted measurement name.
     measurement = "apache_access_log"
-    ## Full path(s) to custom pattern files.
+    
+	## Full path(s) to custom pattern files.
     custom_pattern_files = []
-    ## Custom patterns can also be defined here. Put one pattern per line.
+    
+	## Custom patterns can also be defined here. Put one pattern per line.
```
Comment (danielnelson) [inline plugins/inputs/logparser/README.md]: Use 2 space indention in here

File: plugins/inputs/logparser/grok/grok.go [hunk 3]
```diff
@@ -155,9 +167,16 @@ func (p *Parser) Compile() error {
 		p.Measurement = "logparser_grok"
 	}
 
+	p.loc, err = time.LoadLocation(p.Timezone)
+	if err != nil {
+		log.Printf("D! improper timezone supplied (%s), setting loc to UTC", p.Timezone)
```
Comment (danielnelson) [inline plugins/inputs/logparser/grok/grok.go]: Log this as W!

File: plugins/inputs/logparser/logparser.go [hunk 4]
```diff
@@ -60,23 +63,40 @@ const sampleConfig = `
...
+	## Timezone allows you to provide an override for timestamps that 
+	## don't already include an offset
+	## e.g. 04/06/2016 12:41:45 data one two 5.43µs
+	##
+	## Default: "" which renders UTC
+	## Options are as follows:
+	##   1. Local             -- interpret based on machine localtime
+	##   2. "Canada/Eastern"  -- Unix TZ values like those found in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+	##   3. UTC               -- or blank/unspecified, will return timestamp in UTC
+	timezone = "Canada/Eastern"
```
Comment (danielnelson) [inline plugins/inputs/logparser/logparser.go]: 2 space indention

**Comment by sebito91** [comment]
Updated with requested changes. Apologies for the whitespace issues, new vimrc config :/

**Pull request merged**